src/index.ts

```diff
@@ -54,16 +39,29 @@ const run = async (): Promise<void> => {
     console.log(`Lines percent: ${prData.coverage.lines.percent}`);
     console.log(`Functions percent: ${prData.coverage.functions.percent}`);
     console.log(`Branches percent: ${prData.coverage.branches.percent}`);
-    console.log(prData.coverage.lines.diff);
-    console.log(prData.coverage.functions.diff);
-    console.log(prData.coverage.branches.diff);
+
+    if(prData.coverage.lines.diff || prData.coverage.lines.diff === 0){
```

Comment:
I dont think there is a situation where the second conditional is true and the first is false, unless 0 is not considered as a 'true' value in TS?. If the latter is the case i think we could do this: `if(!!prData.coverage.lines.diff)`